packages/paste-website/src/pages/icons/index.mdx

```diff
@@ -0,0 +1,130 @@
+---
+title: Icons
+description: Our icon system and how it works.
+---
+
+import {graphql} from 'gatsby';
+import {Link} from 'gatsby';
+import {Box} from '@twilio-paste/box';
+import {Button} from '@twilio-paste/button';
+import {PlusIcon} from '@twilio-paste/icons/esm/PlusIcon';
+import {LoadingIcon} from '@twilio-paste/icons/esm/LoadingIcon';
+import Changelog from '@twilio-paste/icons/CHANGELOG.md';
+import {Callout, CalloutTitle, CalloutText} from '../../components/callout';
+import {Grid} from '../../components/grid';
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: {slug: {eq: "/icons/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+<h1>{props.pageContext.frontmatter.title}</h1>
+
+## Guidelines
+
+### About icons
+
+Icons are small graphical representation of a program or a function. They can be used to enhance the
+appearance of a web interface and break up the monotony of text. Paste icons are distributed as both
+SVG files and React components. Our base icon size is set to 24px. Our icon set is still limited,
+we will be adding more as needed. Please [file a feature request](https://github.com/twilio-labs/paste/issues)
+if you need an icon we don't yet have or review our [icon contribution guide](https://docs.google.com/document/d/1a2-KAwU6dOjmcPLRD8zQln_8zS52ti7zziq_JRjUa8E).
+
+#### Accessibility
+
+Using an icon as the only indication of meaning is inaccessible. Different cultures may interpret the same image
+in different ways and visually impaired users may not see the icon. Here are some helpful guidelines to keep in mind:
+
+- Set the `decorative` and `title` props correctly on your icons.
```

Comment:
Is it worth providing details on how to do this? or linking to the part in the doc that tells me how?

Reply:
OK, yeah, there's not really any detail in the doc teaching me how to do this well. I think readers would benefit if we provide a section on _how exactly_ to make a good, accessible icon. What the decorative attribute does, what makes a good title, etc.

Reply:
Let me know if this edit helped https://github.com/twilio-labs/paste/pull/165/commits/9f387ffee77d7589f4f5027b3bf7db290eb1bfd2

Reply:
Better, made a suggestion